app/src/main/java/org/fossasia/pslab/adapters/ControlMainAdapter.java

```diff
@@ -550,14 +597,16 @@ else if (value > 5000)
                     }
                 });
 
-                seekBarControlMain.setMax(5000);
```

Comment:
Because previously range of seekbar was 0-5000 but allowed values were 10-5000. So they were not accurately mapped. 0-4990 -> 10-5000 hope this clears the doubt.